lib/hooks/userspace/hook-express.js

```diff
@@ -83,7 +83,7 @@ function startRootSpanForRequest(req, traceHeader) {
   // we use the path part of the url as the span name and add the full
   // url as a label
   var rootContext = agent.createRootSpanData(req.originalUrl, traceId,
-    parentSpanId);
+    parentSpanId, 3); // Skip 3 frames
```

Comment:
As discussed in person, this philosophy is applied inside the trace agent but doesn't work here in the hooks.